src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php

```diff
@@ -115,11 +115,6 @@ private static function escape(Environment $env, string $s): string
         }
 
         // Twig 3.9
-        if (method_exists(EscaperExtension::class, 'escape')) {
-            return EscaperExtension::escape($env, $s);
-        }
-
-        // to be removed when support for Twig 3 is dropped
-        return twig_escape_filter($env, $s);
+        return EscaperExtension::escape($env, $s);
```

Comment:
This is going to change in 3.10. So maybe we should go straight to 3.10 as a minimum version?

Reply:
I was afraid to do so because 3.10 is not released yet.

Reply:
3.10 is now released :)

Reply:
Updated!